src/svgstore.js

```diff
@@ -53,12 +53,19 @@ function svgstore(options) {
 					var oldDefId = elem.attr('id');
 					var newDefId = id + '_' + oldDefId;
 					elem.attr('id', newDefId);
+
+					/* process use tags */
 					child('use').each(function (i, use) {
 						if (child(use).prop('xlink:href') !== '#' + oldDefId) {
```

Comment:
Since SVG 2, the `xlink:href` attribute is deprecated in favor of simply `href`. We should support both.

Reply:
True, I'll add support of `href` as well.